src/server.rs

```diff
@@ -72,12 +73,35 @@ pub(crate) async fn new(
     })
 }
 
+async fn handler_recip(
+    State(database): State<Arc<Database>>,
+    Path((recipient, order, price)): Path<(String, String, f64)>,
+) -> Json<Response> {
+    let wss = database.rpc().to_string();
+    let mul = database.properties().await.decimals;
+
+    match abcd(database, Some(recipient), order, price).await {
+        Ok(re) => Response::Success(re),
+        Err(error) => Response::Error(Error {
+            wss,
+            mul,
+            version: env!("CARGO_PKG_VERSION").into(),
```

Comment:
Compiled binary does not know this value. Please confirm that this is an issue and we'll add another versioning method later.